webapp/src/components/ImageEditor/store/imageReducer.ts

```diff
@@ -500,6 +512,12 @@ const editorReducer = (state: EditorState, action: any, store: EditorStore): Edi
                 ...state,
                 resizeDisabled: true
             }
+        case actions.CHANGE_ASSET_NAME:
+            tickEvent("change-asset-name");
```

Comment:
sidenote: maybe worth including the type of asset as another field in the `tickEvent`'s data, as it could be interesting to see which types of assets get named most often. Looks like `setTelemetryFunction` is already set up to allow for that anyways

Reply:
I think I'm actually going to remove this tick.... I'll add a different tick elsewhere and think about how to add that info to the general function.